deps.cmake

```diff
@@ -0,0 +1,5 @@
+# Common dependencies for examples
```

Comment:
A recent change to the guidelines now has deps.cmake inside the examples directory rather than at the top-level, and then tests can include this via `../../examples/deps.cmake`